docs/source/developer/media.rst

```diff
@@ -0,0 +1,40 @@
+Media Application
```

Comment:
Change to just "Media", the application on other pages is referring to custom Django applications.